### PR TITLE
news: Use Markdown and fix formatting of GSoC 2013 post

### PR DESCRIPTION
**Pull request by Holzhaus**

### CODE REVIEW

**Review by daschuer — APPROVED** [review]
LGTM

File: content/news/2013-04-08-calling-all-students-hack-on-mixxx-in-google-summer-of-code-2013.md [hunk 1]
```diff
@@ -4,4 +4,20 @@ date: 2013-04-08 19:16:00
...
+[Google Summer of Code 2013]({static}/images/news/image00.jpg)
```
Comment (Swiftb0y) [inline content/news/2013-04-08-calling-all-students-hack-on-mixxx-in-google-summer-of-code-2013.md:7]:
```suggestion
![Google Summer of Code 2013]({static}/images/news/image00.jpg)
```

**Review by Swiftb0y — CHANGES_REQUESTED** [review]
LGTM otherwise

File: content/news/2013-04-08-calling-all-students-hack-on-mixxx-in-google-summer-of-code-2013.md (hunk 1 above)
Comment (Holzhaus) [reply content/news/2013-04-08-calling-all-students-hack-on-mixxx-in-google-summer-of-code-2013.md:7]: Too late, but I fixed it and pushed to the website branch directly.

**Pull request merged**